changelog/pending/20251113--cli--expose-all-command-flags-as-env-vars-e-g-refresh-can-now-be-set-with-pulumi_refresh.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli
+  description: Expose all command flags as env vars (e.g. --refresh can now be set with PULUMI_REFRESH)
```

Comment:
```suggestion
  description: Expose all command flags as env vars (e.g. --refresh can now be set with PULUMI_OPTION_REFRESH)
```

I think this is the naming pattern we decided for?

Reply:
WOW that is embarrassing, thank you